pkgs/by-name/an/anydesk/package.nix

```diff
@@ -123,7 +129,8 @@ stdenv.mkDerivation (finalAttrs: {
           lsb-release
           pciutils
         ]
-      }
+      } \
+      --prefix GDK_BACKEND : x11
```

Comment:
```suggestion
      --prefix GDK_BACKEND : x11 \
      --set GTK_THEME Adwaita \
```

Reply:
we cannot ensure that that theme is always installed right?

Reply:
If I'm not mistaken it ships with `gtk3`, or with some very common dependency that the user most likely already has, not sure how to check it. Even if the theme is not found (eg. if I set `GTK_THEME=foobar`, it will fall back to an existing theme, Adwaita for me), For reference, another package setting `GTK_THEME`: https://github.com/NixOS/nixpkgs/blob/b5fce003f4f16f8409703cb140656da75015bb99/pkgs/applications/networking/remote/vmware-horizon-client/default.nix#L30

Reply:
Good evidence, thanks for the research

I've force-pushed the commit with the suggested patch

Reply:
> we cannot ensure that that theme is always installed right?

Tested with a non-existent theme name: in this case GTK will falls back to the  default theme, so it does not seem to be a problem (if it ever occurs)